e2e/solid-start/dev-ssr-styles/src/styles/code-split-route.module.css

```
@@ -0,0 +1,6 @@
+.styledBox {
+  --solid-code-split-style-marker: present;
+  background-color: #3b82f6;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)